src/chains/ethereum/ethereum/src/forking/handlers/http-handler.ts

```diff
@@ -112,6 +112,7 @@ export class HttpHandler extends BaseHandler implements Handler {
       const postData = `${JSONRPC_PREFIX}${this.id++},${key.slice(1)}`;
       this.headers["content-length"] = postData.length;
 
+      // @ts-ignore
       const req = this._request(requestOptions);
```

Comment:
I think node's types updated and it doesn't like the polyfill we use for the `abortSignal` anymore.

Reply:
removed this change in https://github.com/trufflesuite/ganache/pull/4459